Plugins/Source/KISAddonCursor.cs

```diff
@@ -89,6 +82,17 @@ sealed class KISAddonCursor : MonoBehaviour {
     Cursor.visible = false;
   }
 
+  void Awake() {
+    hintWindowStyle = new GUIStyle {
```

Comment:
Awake () is run as soon as the script is loaded (according to Unity docs) and one of the APIs used on that line threw an error as not being allowed to run in ctors/initializers.

Can change the member to instance but then if there's any chance of the instance getting destructed might also need to manually Dispose of the resource. 

Conversely since there's only once we load the script and set the static even though we may have multiple instances they'll all share the member keeping the pre-1.2 logic. I don't know KIS well enough to know if this is a singleton or not.